components/mqtt.rst

```diff
@@ -137,10 +139,93 @@ Configuration options:
 -  **retain** (*Optional*, boolean): If the published message should
    have a retain flag on or not. Defaults to ``true``.
 
-.. _mqtt-using_with_home_assistant:
 
-Using with Home Assistant
--------------------------
+.. _mqtt-device_discovery:
+
+MQTT device discovery
+---------------------
+
+The ESPHome device will respond to the following MQTT topics if `mqtt.discover_ip` is enabled.
```

Comment:
All inline code should be double backticks, single just makes it italic.
```suggestion
The ESPHome device will respond to the following MQTT topics if ``mqtt.discover_ip`` is enabled.
```

Reply:
changed